pkgs/top-level/all-packages.nix

```diff
@@ -8990,6 +8990,8 @@ with pkgs;
 
   kakoune-cr = callPackage ../tools/misc/kakoune-cr { crystal = crystal_1_2; };
 
+  kamp = callPackage ../tools/misc/kamp/package.nix { };
```

Comment:
Move this package to `pkgs/by-name` - new packages should be created here! [Reference](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#readme)